### PR TITLE
CON-208: initiate PoSv1 governance section

### PR DESCRIPTION
**Pull request by DannyS03**
- Adds process and key terms for governance.
- References forum.

### CODE REVIEW

File: src/data/features.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const firstRow = [
   {
       title: "Governance",
       status: "Beginner: Docs Coming Soon",
```
Comment (ashucoder9) [inline src/data/features.js:18]:
```suggestion
      status: "Beginner",
```

File: docs/maintain/govern/governance-posv1.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+sidebar_label: PoS v1 Governance
+description: Common governance terms and processes for the Polygon PoS v1 blockchain.
+keywords:
+  - docs
+  - matic
+  - polygon
+  - DAO
+  - governance
+slug: governance-posv1
+image: https://matic.network/banners/matic-network-16x9.png
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:13]:
```suggestion
image: https://wiki.polygon.technology/img/polygon-wiki.png
```

File: docs/maintain/govern/governance-posv1.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+<!-- This page is a WIP -->
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:16]:
```suggestion
```

File: docs/maintain/govern/governance-posv1.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+This page includes common governance terms and processes for the Polygon PoS v1 blockchain.
+
+## Hemidall
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:20]:
```suggestion
## Heimdall
```

File: docs/maintain/govern/governance-posv1.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+|-------------------|-------------------|-----------------------------------------------|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Software upgrades | Both              | PIP proposed via community forum (recommeded) | Rough consensus        | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet                                                                                                                                                                                                     |
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:24]:
```suggestion
| Software upgrades | Both              | PIP proposed via community forum (recommended) | Rough consensus        | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet                                                                                                                                                                                                     |
```

File: docs/maintain/govern/governance-posv1.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+| Consensus rules   | Hardfork only     | PIP proposed via community forum (recommeded) | Rough consensus        | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet                                                                                                                                                                                                     |
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:25]:
```suggestion
| Consensus rules   | Hardfork only     | PIP proposed via community forum (recommended) | Rough consensus        | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet                                                                                                                                                                                                     |
```

File: docs/maintain/govern/governance-posv1.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+| [Governance module](https://forum.polygon.technology/t/overview-of-governance-on-matic/8191) | Both              | PIP proposed via community forum (recommeded) | Governance module vote | Governance module:  1. Proposal submission: Validators can submit proposals with a deposit. Once the minimum deposit is reached, the proposal enters the voting period. Validators that deposited on proposals can recover their deposits once the proposal is rejected or accepted. 2. Vote: Validators can vote on proposals that reached MinDeposit 3. Once the voting parameters needed to pass the proposal are met, the change is simultaneously carried out across the network |
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:26]:
```suggestion
| [Governance module](https://forum.polygon.technology/t/overview-of-governance-on-matic/8191) | Both              | PIP proposed via community forum (recommended) | Governance module vote | Governance module:  1. Proposal submission: Validators can submit proposals with a deposit. Once the minimum deposit is reached, the proposal enters the voting period. Validators that deposited on proposals can recover their deposits once the proposal is rejected or accepted. 2. Vote: Validators can vote on proposals that reached MinDeposit 3. Once the voting parameters needed to pass the proposal are met, the change is simultaneously carried out across the network |
```

File: docs/maintain/govern/governance-posv1.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+|-------------------|-------------------|----------------------------------------------------------------------------------------------------|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Software upgrades | Both              | PIP proposed via community forum (recommeded)                                                      | Rough consensus   | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet |
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:32]:
```suggestion
| Software upgrades | Both              | PIP proposed via community forum (recommended)                                                      | Rough consensus   | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet |
```

File: docs/maintain/govern/governance-posv1.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+| Consensus rules   | Hardfork only     | Upgrades are made in line with the  [geth](https://github.com/ethereum/go-ethereum) implementation | Rough consensus   | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet |
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:33]:
```suggestion
| Consensus rules   | Hardfork only     | Upgrades are made in line with the  [geth](https://github.com/ethereum/go-ethereum) implementation | Rough consensus   | Rough consensus:  1. Development and testing on local dev machines 2. Multi-node testing on devnet  3. Rolling out the release on a few Mumbai nodes (testnet) 4. Rolling out the release on a few Mainnet nodes 5. Full roll-out on Mumbai (testnet) 6. Full roll-out on Mainnet |
```

File: docs/maintain/govern/governance-posv1.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+## Definitions:
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:35]:
```suggestion
## Definitions
```

File: docs/maintain/govern/governance-posv1.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+### Softfork
+
+This type of change is backward-compatible with the pre-fork blocks. This protocol change does
+not require nodes to upgrade before a deadline. Therefore multiple versions of the node software can
+be running at once and be able to validate transactions.
+
+### Rough Consensus
+
+Defined as the 'dominant view' of a group as determined by the current consensus framework. In the
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:61]:
```suggestion
Defined as the 'dominant view' of a group as determined by the current consensus framework. 
```

File: docs/maintain/govern/governance-posv1.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+Without a vote that can carry out a synchronous update across the network, the 'dominant view.'
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:62]:
```suggestion
Without a vote that can carry out a synchronous update across the network, the 'dominant view'
```

File: docs/maintain/govern/governance-posv1.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+along with a deposit containing the proposed changes. Each validator tallies votes cast by validators.
+Once the defined voting parameters are met, each validator automatically updates itself
+with the proposal data.
+
+The current voting parameters (denominated in staked MATIC):
+- Quorum: 33.4%
+- Threshold: 50%
+- Veto: 33.4%
+
+A list of the changeable parameters by the Heimdall governance module is available here.
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:80]: I guess there is a missing link at the end 'here'.

File: docs/maintain/govern/governance-posv1.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+
+### Polygon Improvement Proposal (PIP)
+
+The PIP process, as defined in PIP-1, outlined a preliminary approach for allowing the community
+to put forward-protocol upgrades that improve the network.
+
+This takes inspiration from the PEP process, which has become a standard used for open source
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:87]:
```suggestion
This takes inspiration from the [PEP process](https://peps.python.org/pep-0001/), which has become a standard used for open source
```

File: docs/maintain/govern/governance-posv1.md [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+projects, including Ethereum and Bitcoin.
+
+Due to the permissionless nature of governance on the Polygon POS chain, ⅔+1 of the total stake can
```
Comment (ashucoder9) [inline docs/maintain/govern/governance-posv1.md:90]:
```suggestion
Due to the permissionless nature of governance on the Polygon PoS chain, ⅔+1 of the total stake can
```

**Review by ashucoder9 — APPROVED** [review]

File: docs/maintain/govern/governance-posv1.md (hunk 13 above)
Comment (DannyS03) [inline docs/maintain/govern/governance-posv1.md:80]:
```suggestion
A list of the changeable parameters by the Heimdall governance module is available [here](https://github.com/maticnetwork/heimdall/blob/develop/auth/types/params.go).
```

**Pull request merged**